src/Common/src/System/Net/Logging/SocketsEventSource.cs

```diff
@@ -0,0 +1,268 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics.Tracing;
+using System.Globalization;
+using System.Net.Sockets;
+using System.Runtime.InteropServices;
+
+namespace System.Net
+{
+    //TODO: If localization resources are not found, logging does not work.
```

Comment:
Please add more context why this is relevant here (this EventSource is included in multiple projects? Which ones?).
Also, please add an issue reference.